### ISSUE TITLE
Syntastic: No Checkers?

### ISSUE THREAD

**Issue by auxym** [zah/nim.vim#94]
Upon installing this plugin and opening a nim file, I don't seem to get any syntax checking via syntastic. Running `:SyntasticInfo` yields the following:

```
Syntastic version: 3.9.0-23 (Vim 800, Neovim, Linux)                                                                          
Info for filetype: nim                                                                                                        
Global mode: active                                                                                                           
Filetype nim is active                                                                                                        
The current file will be checked automatically                                                                                
Available checkers: -                                                                                                         
Currently enabled checkers: -  
```

So syntastic reports no available checkers. Is there any setup required here?

**Comment by auxym** [comment]
I got it working by adding the following to my init.vim (in neovim):

```viml
let g:syntastic_<filetype>_checkers = ['<checker-name>']
```

Perhaps this should be added in the autoload script? Or at least mentioned in the README.

**Comment by zah** [comment]
Fixed in f16af98
